test/extended/build_context_dir_test.go

```diff
@@ -0,0 +1,76 @@
+// +build integration,extended,!no-etcd,!no-docker,docker
```

Comment:
I think you may want to leave `integration,extended` since you're passing those with `OS_TEST_TAGS`